### PR TITLE
Removed notes, final commit.

### PR DESCRIPTION
**Pull request by olliejth**
Removed notes from the fake_data.sql file, other than that no changes since #115

### CODE REVIEW

**Comment by AlexOBarnes** [comment]
This closes #125

File: database/fake_data.sql [hunk 1]
```diff
@@ -45,35 +45,27 @@ VALUES
...
-('NASA_apod', 'https://science.nasa.gov/wp-content/uploads/2023/09/Carina_Nebula-1.jpeg', '2024-10-02', 2, 2),
-('NASA_apod', 'https://science.nasa.gov/wp-content/uploads/2023/09/Carina_Nebula-1.jpeg', '2024-10-03', 3, 3),
-('ASTRO_moon', 'https://upload.wikimedia.org/wikipedia/commons/thumb/e/e1/FullMoon2010.jpg/1200px-FullMoon2010.jpg', '2024-10-04', 4, 4),
-('ASTRO_moon', 'https://upload.wikimedia.org/wikipedia/commons/thumb/e/e1/FullMoon2010.jpg/1200px-FullMoon2010.jpg', '2024-10-05', 5, 5),
+('NASA_apod', 'https://science.nasa.gov/wp-content/uploads/2023/09/Carina_Nebula-1.jpeg', '2024-10-01', NULL, 1),
+('NASA_apod', 'https://science.nasa.gov/wp-content/uploads/2023/09/Carina_Nebula-1.jpeg', '2024-10-02', NULL, 2),
+('NASA_apod', 'https://science.nasa.gov/wp-content/uploads/2023/09/Carina_Nebula-1.jpeg', '2024-10-03', NULL, 3),
+('ASTRO_moon', 'https://upload.wikimedia.org/wikipedia/commons/thumb/e/e1/FullMoon2010.jpg/1200px-FullMoon2010.jpg', '2024-10-04', NULL, 4),
+('ASTRO_moon', 'https://upload.wikimedia.org/wikipedia/commons/thumb/e/e1/FullMoon2010.jpg/1200px-FullMoon2010.jpg', '2024-10-05', NULL, 5),
```
Comment (AlexOBarnes) [inline database/fake_data.sql:52]: Thanks for updating this

File: database/fake_data.sql [hunk 2]
```diff
@@ -45,35 +45,27 @@ VALUES
...
-('ASTRO_moon', 'https://upload.wikimedia.org/wikipedia/commons/thumb/e/e1/FullMoon2010.jpg/1200px-FullMoon2010.jpg', '2024-10-10', 10, 10),
+('NASA_apod', 'https://science.nasa.gov/wp-content/uploads/2023/09/Carina_Nebula-1.jpeg', '2024-10-08', NULL, 8),
+('NASA_apod', 'https://science.nasa.gov/wp-content/uploads/2023/09/Carina_Nebula-1.jpeg', '2024-10-09', NULL, 9),
+('ASTRO_moon', 'https://upload.wikimedia.org/wikipedia/commons/thumb/e/e1/FullMoon2010.jpg/1200px-FullMoon2010.jpg', '2024-10-10', NULL, 10),
 ('ASTRO_star', 'https://dq0hsqwjhea1.cloudfront.net/Interactive-Sky-Chart-600px.jpg', '2024-10-11', 11, 11),
-('NASA_apod', 'https://science.nasa.gov/wp-content/uploads/2023/09/Carina_Nebula-1.jpeg', '2024-10-12', 12, 12),
-('ASTRO_moon', 'https://upload.wikimedia.org/wikipedia/commons/thumb/e/e1/FullMoon2010.jpg/1200px-FullMoon2010.jpg', '2024-10-13', 13, 13),
+('NASA_apod', 'https://science.nasa.gov/wp-content/uploads/2023/09/Carina_Nebula-1.jpeg', '2024-10-12', NULL, 12),
+('ASTRO_moon', 'https://upload.wikimedia.org/wikipedia/commons/thumb/e/e1/FullMoon2010.jpg/1200px-FullMoon2010.jpg', '2024-10-13', NULL, 13),
```
Comment (AlexOBarnes) [inline database/fake_data.sql:60]: Good catch

**Review by AlexOBarnes — APPROVED** [review]
This is a really useful update

File: database/fake_data.sql [hunk 3]
```diff
@@ -45,35 +45,27 @@ VALUES
 
 INSERT INTO image (image_name, image_url, image_date, region_id, constellation_id)
 VALUES
-('NASA_apod', 'https://science.nasa.gov/wp-content/uploads/2023/09/Carina_Nebula-1.jpeg', '2024-10-01', 1, 1),
-('NASA_apod', 'https://science.nasa.gov/wp-content/uploads/2023/09/Carina_Nebula-1.jpeg', '2024-10-02', 2, 2),
-('NASA_apod', 'https://science.nasa.gov/wp-content/uploads/2023/09/Carina_Nebula-1.jpeg', '2024-10-03', 3, 3),
-('ASTRO_moon', 'https://upload.wikimedia.org/wikipedia/commons/thumb/e/e1/FullMoon2010.jpg/1200px-FullMoon2010.jpg', '2024-10-04', 4, 4),
-('ASTRO_moon', 'https://upload.wikimedia.org/wikipedia/commons/thumb/e/e1/FullMoon2010.jpg/1200px-FullMoon2010.jpg', '2024-10-05', 5, 5),
+('NASA_apod', 'https://science.nasa.gov/wp-content/uploads/2023/09/Carina_Nebula-1.jpeg', '2024-10-01', NULL, 1),
```
Comment (CalebMathers) [inline database/fake_data.sql:48]: Good to have NULL values as these aren't linked to a region

File: database/fake_data.sql [hunk 4]
```diff
@@ -45,35 +45,27 @@ VALUES
...
+('NASA_apod', 'https://science.nasa.gov/wp-content/uploads/2023/09/Carina_Nebula-1.jpeg', '2024-10-17', NULL, 12),
+('ASTRO_moon', 'https://upload.wikimedia.org/wikipedia/commons/thumb/e/e1/FullMoon2010.jpg/1200px-FullMoon2010.jpg', '2024-10-18', NULL, 45),
+('NASA_apod', 'https://science.nasa.gov/wp-content/uploads/2023/09/Carina_Nebula-1.jpeg', '2024-10-19', NULL, 88),
 ('ASTRO_star', 'https://dq0hsqwjhea1.cloudfront.net/Interactive-Sky-Chart-600px.jpg', '2024-10-20', 12, 36),
-('ASTRO_moon', 'https://upload.wikimedia.org/wikipedia/commons/thumb/e/e1/FullMoon2010.jpg/1200px-FullMoon2010.jpg', '2024-10-21', 15, 55),
-('NASA_apod', 'https://science.nasa.gov/wp-content/uploads/2023/09/Carina_Nebula-1.jpeg', '2024-10-22', 20, 8),
-('ASTRO_star', 'https://dq0hsqwjhea1.cloudfront.net/Interactive-Sky-Chart-600px.jpg', '2024-10-23', 25, 73),
-('ASTRO_moon', 'https://upload.wikimedia.org/wikipedia/commons/thumb/e/e1/FullMoon2010.jpg/1200px-FullMoon2010.jpg', '2024-10-24', 30, 21),
-('NASA_apod', 'https://science.nasa.gov/wp-content/uploads/2023/09/Carina_Nebula-1.jpeg', '2024-10-25', 35, 50);
```
Comment (CalebMathers) [inline database/fake_data.sql:72]: Shouldn't be any issues with invalid region id now bosh

File: database/fake_data.sql [hunk 5]
```diff
@@ -162,7 +154,7 @@ VALUES
 ('2024-10-01 06:56:00', '2024-10-01 19:11:00', 27),
 ('2024-10-01 06:57:00', '2024-10-01 19:12:00', 28),
 ('2024-10-01 06:58:00', '2024-10-01 19:13:00', 29),
-('2024-10-01 06:59:00', '2024-10-01 19:14:00', 30),
+('2024-10-01 06:59:00', '2024-10-01 19:14:00', 92),
```
Comment (CalebMathers) [inline database/fake_data.sql:157]: That covers all counties wooo

**Review by CalebMathers — APPROVED** [review]

File: database/fake_data.sql (hunk 5 above)
Comment (MurasatoNaoya) [reply database/fake_data.sql:157]: All counties are accounted for, LGTM.

File: database/fake_data.sql (hunk 3 above)
Comment (MurasatoNaoya) [reply database/fake_data.sql:48]: As discussed, NULL values are accepted by the schema and actually necessary so the expected numbers are passed in. Needed addition, LGTM.

**Review by MurasatoNaoya — APPROVED** [review]

**Pull request merged**